### DISCUSSION TITLE
Everyone wants to go back to the old feed page

### DISCUSSION THREAD

**Discussion by webfansplz** [community/community#66558] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I've always believed that GitHub listens to its users' feedback attentively and is proactive in making changes.

I believe the extensive discussions indicate that people do not want the new feed page.

I'm curious to know if the official team has plans to revert to the old feed page or at least provide a switch option. 🙏

**Comment by Fox2Code** [comment]
Check that to revert manually:

https://github.com/orgs/community/discussions/66188
